general_tools/log.py

```diff
@@ -50,3 +50,12 @@ def get_logger(name='NTHU OJ'):
     logger.addHandler(ch)
 
     return logger
+
+
+def get_client_ip(request):
```

Comment:
I want to open discussion about this kind of code. 
Now we have ```general_tools``` (In fact, I like ```utils``` better).
We need to consider the possible usages of these functions.
For this case, I think the best position may be ```network_info.py``` or ```user_info.py```.
(Just a example, you can discuss about this)